emission/tests/storageTests/TestSectionQueries.py

```diff
@@ -100,7 +100,6 @@ def testCleaned2InferredSectionList(self):
         # Total = 25 = 10 (UUID1) + 15 (UUID2)
         curr_predicted_entries = esds.cleaned2inferred_section_list(section_user_list)
         self.assertEqual(len(curr_predicted_entries), len(sections_entries))
-        print(curr_predicted_entries)
```

Comment:
why did you remove this?

Reply:
Same as above; removing this as I had added them earlier.